src/code/PreRender.c

```diff
@@ -408,15 +693,15 @@ void PreRender_ApplyFilters(PreRender* this) {
     } else {
         this->unk_4D = 1;
```

Comment:
To me looks like `unk_4D` seems to be a `FilterState` with an enum like
```c
typedef enum {
    /* 0 */ PRERENDER_FILTER_STATE_NONE,
    /* 1 */ PRERENDER_FILTER_STATE_PROCESS,
    /* 2 */ PRERENDER_FILTER_STATE_DONE
} PrerenderFilterState;
```
(I'm not too attached to PROCESS, but i like how it mirrors `MotionBlurStatus`, `PictoPhotoState`, `TransitionTileState`, and `PauseBgPreRenderState`)